samcli/lib/remote_invoke/remote_invoke_executors.py

```diff
@@ -4,13 +4,23 @@
 import json
 import logging
 from abc import ABC, abstractmethod
+from enum import Enum
 from io import TextIOWrapper
 from pathlib import Path
 from typing import Any, Callable, List, Optional, Union, cast
 
 LOG = logging.getLogger(__name__)
 
 
+class RemoteInvokeOutputFormat(Enum):
+    """
+    Types of output formats used to by remote invoke
+    """
+
+    DEFAULT = "default"
```

Comment:
This feels like it could be more pythonic.

Reply:
Discussed offline this is not a blocker. Keeping this enum consistent with the other enums we have where the string is assigned directly to the constant.